app/views/teachers/_form.html.erb

```diff
@@ -10,12 +10,42 @@
       </ul>
     </div>
   <% end %>
+  <div class="outer-container">
+    <div class="container">
```

Comment:
How come you have these `<div class="container">` elements? If you're doing it for styling purposes, could you instead style each `div.field`?